Sources/TreeSitterClient/TreeSitter+Extensions.swift

```diff
@@ -15,12 +15,8 @@ extension InputEdit {
             return nil
         }
 
-        guard
-            let startPoint = transformer(startLocation),
-            let newEndPoint = transformer(newEndLocation)
-        else {
-            return nil
-        }
+        let startPoint = transformer?(startLocation) ?? .zero
+        let newEndPoint = transformer?(newEndLocation) ?? .zero
```

Comment:
This is a subtle change. I'd like to make a clear distinction between the `transformer` being nil, and it being non-nil and returning a nil value.

Reply:
Update the code to accommodate the mentioned change